helmfile.d/values/userCRDs/jaeger/jaeger.yaml.gotmpl

```
@@ -0,0 +1,320 @@
+      - watch
+    - apiGroups:
+      - apps
+      resources:
+      - deployments/status
+      verbs:
+      - get
+      - patch
+      - update
+    - apiGroups:
+      - autoscaling
+      resources:
+      - horizontalpodautoscalers
+      verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+    - apiGroups:
+      - batch
+      resources:
+      - cronjobs
+      - jobs
+      verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+    - apiGroups:
+      - console.openshift.io
+      resources:
+      - consolelinks
+      verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+    - apiGroups:
+      - coordination.k8s.io
+      resources:
+      - leases
+      verbs:
+      - create
+      - get
+      - list
+      - update
+    - apiGroups:
+      - ""
+      resources:
+      - configmaps
+      - persistentvolumeclaims
+      - pods
+      - secrets
+      - serviceaccounts
+      - services
+      - services/finalizers
+      verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+    - apiGroups:
+      - ""
+      resources:
+      - namespaces
+      verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+    - apiGroups:
+      - ""
+      resources:
+      - namespaces/status
+      verbs:
+      - get
+      - patch
+      - update
+    - apiGroups:
+      - extensions
+      resources:
+      - ingresses
+      verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+    - apiGroups:
+      - image.openshift.io
+      resources:
+      - imagestreams
+      verbs:
+      - get
+      - list
+      - watch
+    - apiGroups:
+      - jaegertracing.io
+      resources:
+      - jaegers
+      verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+    - apiGroups:
+      - jaegertracing.io
+      resources:
+      - jaegers/finalizers
+      verbs:
+      - update
+    - apiGroups:
+      - jaegertracing.io
+      resources:
+      - jaegers/status
+      verbs:
+      - get
+      - patch
+      - update
+    - apiGroups:
+      - kafka.strimzi.io
+      resources:
+      - kafkas
+      - kafkausers
+      verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+    - apiGroups:
+      - logging.openshift.io
+      resources:
+      - elasticsearch
+      verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+    - apiGroups:
+      - logging.openshift.io
+      resources:
+      - elasticsearches
+      verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
```

Nit: would take up less space if these were added like this:
```suggestion
    - apiGroups:
      - logging.openshift.io
      resources:
      - elasticsearch
      - elasticsearches
      verbs:
      - create
      - delete
      - get
      - list
      - patch
      - update
      - watch
```
Since both resources come from the same apigroup `logging.openshift.io` and has the same verbs.